rpmlb/builder/base.py

```diff
@@ -11,6 +12,17 @@
 LOG = logging.getLogger(__name__)
 
 
+#: Regular expression for finding macro definitions
+MACRO_REGEX = re.compile(
+    r'''^
+    %global\s+                   # beginning of definition
+    (?P<name>[^\s]+)\s+          # one-word name
+    (?P<value>(?:.|(?<=\\)\n)+)  # value including spaces and escaped newlines
+    $''',
+    flags=re.MULTILINE | re.VERBOSE,
+)
```

Comment:
Why do you want to include "and escaped newlines"?
Matching `^%global\s+foo\s+var\s+$` is good enough?